docs/content/guides/6.multistore/3.patterns/2.design/4.different-code-per-store.md

```diff
@@ -1,8 +1,208 @@
 ---
-title: Different markups for individual components
+title: Different Markups for Individual Components
 layout: default
 navigation:
   icon: tabler:number-1-small
 ---
 
-# Different markups for individual components
+# Different Markups for Individual Components
+
+> **Prerequisite:** Familiarity with [Tooling & Concepts → File-Based Inheritance](../2.file-based-inheritance.md) is required to understand file override mechanisms and inheritance principles.
```

Comment:
Usually we write Prerequisite as a `## Prerequisites` section (you can find examples by searching the code base for the word Prerequisites)

Reply:
Fixed